src/metabase/util/embed.clj

```diff
@@ -0,0 +1,26 @@
+(ns metabase.util.embed
+  (:require [hiccup.core :refer [html]]
+            [metabase.util.urls :as urls]))
+
+(defn- oembed-link
+  [url format content-type]
+  (html [:link {:rel   "alternate"
+                :type  content-type
+                :href  (urls/oembed-url url format)
+                :title "Metabase"}]))
+
+(defn- embedly-meta
+  []
```

Comment:
If there's no docstring you usually just put the arglist on the same line as the function name.

```clojure
(defn- embedly-meta []
  ...)
```